src/lib/utils/unique_random_name.cpp

```diff
@@ -0,0 +1,9 @@
+#include "unique_random_name.hpp"
+
+namespace opossum {
+
+std::string unique_random_name() {
```

Comment:
Why is this in `src/lib`, not in `src/test`?

Reply:
Because I thought it’s a utility. Do we have utility / helper files for tests?

Reply:
Not yet. But there is nothing that keeps us from introducing a `utils` folder for tests.

Reply:
There is a folder `src/test/utils` which contains the tests for files under `src/utils`

Reply:
How ambiguous. :D 
Maybe `src/lib/testing_utils.hpp`?
One reason not to have it in `src/lib` is that it would require the code to be tested. The second reason is that the actual code does not need that method.

Reply:
d'oh, yes.